src/utils/azure.ts

```diff
@@ -26,35 +30,77 @@ export const agid: IAzureConfig = {
   cosmosNotificationStatusContainer: "notification-status",
   cosmosProfilesContainer: "profiles",
   cosmosServicesContainer: "services",
+  cosmosSenderServicesContainer: "sender-services",
+  cosmosUserDataProcessingContainer: "user-data-processing",
+  cosmosUserBonusesContainer: "user-bonuses",
   cosmosName: "agid-cosmosdb-test",
   resourceGroup: "agid-rg-test",
   storageMessagesContainer: "message-content",
+  storageBonusRedeemedContainer: "",
   storageName: "agidstoragetest"
 };
 
 export const dev: IAzureConfig = {
-  configName: "io-dev-aks-k8s-01",
-  cosmosDatabaseName: "io-dev-sqldb-db-01",
+  configName: "io-d",
+  cosmosDatabaseName: "raimondo",
   cosmosMessagesContainer: "messages",
   cosmosMessageStatusContainer: "message-status",
   cosmosNotificationContainer: "notifications",
   cosmosNotificationStatusContainer: "notification-status",
   cosmosProfilesContainer: "profiles",
   cosmosServicesContainer: "services",
-  cosmosName: "io-dev-cosmosdb-01",
-  resourceGroup: "io-dev-rg",
+  cosmosSenderServicesContainer: "sender-services",
+  cosmosUserDataProcessingContainer: "user-data-processing",
+  cosmosUserBonusesContainer: "user-bonuses",
+  cosmosName: "io-d-cosmos-free",
+  resourceGroup: "io-d-rg-common",
   storageMessagesContainer: "message-content",
-  storageName: "iodevsaappdata"
+  storageBonusRedeemedContainer: "redeemed-test",
+  storageName: "devpasqualesa"
```

Comment:
?

Reply:
fixed and created `iodsttest` in dev environment